flutter/lib/mobile/pages/remote_page.dart

```diff
@@ -531,7 +522,18 @@ class _RemotePageState extends State<RemotePage> {
                       controller: _textController,
                       // trick way to make backspace work always
                       keyboardType: TextInputType.multiline,
-                      onChanged: handleSoftKeyboardInput,
+                      onChanged: (newValue) async {
+                        if (isIOS) {
+                          // fix: TextFormField onChanged event triggered multiple times when Korean input
+                          // https://github.com/rustdesk/rustdesk/pull/9644
+                          await Future.delayed(const Duration(milliseconds: 10));
+
+                          if (newValue != _textController.text) return;
+                          return _handleIOSSoftKeyboardInput(_textController.text);
+                        } else {
+                          return _handleNonIOSSoftKeyboardInput(newValue);
+                        }
+                      },
```

Comment:
I think modifying in handleSoftKeyboardInput is better.

Reply:
https://github.com/rustdesk/rustdesk/actions/runs/11364668257/job/31611171027 You did not test after change?

Reply:
@rustdesk Sorry, there is indeed no test, the position of async is wrong, can you revert this merge, I will resubmit the PR.